staging/src/k8s.io/apiextensions-apiserver/examples/client-go/README.md

```diff
@@ -2,18 +2,29 @@
 
 **Note:** CustomResourceDefinition is the successor of the deprecated ThirdPartyResource.
 
-This particular example demonstrates how to perform basic operations such as:
+This particular example demonstrates how to generate a client for CustomResources using [`k8s.io/code-generator`](https://github.com/kubernetes/code-generator). The clientset can
+be generated using the `./hack/update-codegen.sh` script.
 
-* How to register a new custom resource (custom resource type) using a CustomResourceDefinition
-* How to create/get/list instances of your new resource type (update/delete/etc work as well but are not demonstrated)
-* How to setup a controller on resource handling create/update/delete events
+The `update-codegen` script will automatically generate the following files and
+directories:
 
-## Running
+* `pkg/apis/samplecontroller/v1alpha1/zz_generated.deepcopy.go`
```

Comment:
should we change the api name? Is it used anywhere else?

Reply:
copy-paste error. will change this.